Extension/src/telemetry.ts

```diff
@@ -139,6 +139,15 @@ export function logLanguageServerEvent(eventName: string, properties?: { [key: s
     sendTelemetry();
 }
 
+export async function showStatusBarIntellisenseIndicator(): Promise<boolean> {
+    if (new CppSettings().experimentalFeatures) {
+        return true;
+    }
+    const experimentationService: IExperimentationService | undefined = await getExperimentationService();
+    const useNewUI: boolean | undefined = experimentationService?.getTreatmentVariable<boolean>("vscode", "showStatusBarIntellisenseIndicator");
```

Comment:
same here

Reply:
This needs to match whatever Alex configured for the treatment variable. It's not publicly facing, so it doesn't matter in the end.